deploy/6-deploy-weETH-oracle.ts

```diff
@@ -0,0 +1,46 @@
+import { ethers } from "hardhat";
+import { DeployFunction } from "hardhat-deploy/dist/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { ADDRESSES, assets } from "../helpers/deploymentConfig";
+
+const func: DeployFunction = async ({ getNamedAccounts, deployments, network }: HardhatRuntimeEnvironment) => {
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  const resilientOracle = await ethers.getContract("ResilientOracle");
+  const proxyOwnerAddress = network.live ? ADDRESSES[network.name].timelock : deployer;
+
+  let { EtherFiLiquidityPool, weETH, eETH } = ADDRESSES[network.name];
+
+  if (!EtherFiLiquidityPool) {
+    // deploy mock liquidity pool
+    await deploy("MockEtherFiLiquidityPool", {
+      from: deployer,
+      contract: "MockEtherFiLiquidityPool",
+      args: [],
+      log: true,
+      autoMine: true, // speed up deployment on local network (ganache, hardhat), no effect on live networks
+      skipIfAlreadyDeployed: true,
+    });
+
+    const mockEtherFiLiquidityPool = await ethers.getContract("MockEtherFiLiquidityPool");
+    EtherFiLiquidityPool = mockEtherFiLiquidityPool.address;
+    await mockEtherFiLiquidityPool.transferOwnership(proxyOwnerAddress);
+  }
+
```

Comment:
I would also deploy the mocks tokens for `weETH` and `eETH` if the address is not available in the `ADDRESSES` mapping (for example on hardhat). /cc @coreyar

Reply:
I highly recommend not using address mappings at all and instead fetching deployments

Reply:
We can structure deployment scripts into 3 types of scripts for each deployment, (mocks, deploy, setup)
for example there could be 
deploy/6-mocks-weETH-oracle.ts ( mocks wouldn't run in the right order but a name like `dependencies` would ) 
deploy/6-deploy-weETH-oracle.ts
deploy/6-setup-weETH-oracle.ts

Where setup receives an id so that it is only run once and mocks isn't deployed on live networks ( it would deploy the token contracts Jesus mentions) deploy only deploys contracts and setup doesn't deploy and only calls methods on contracts

Reply:
VEN-2547. We can leave this as it is in this PR, and review it in the new task.